nil/tests/cli_call/cli_call_test.go

```diff
@@ -33,21 +32,14 @@ func (s *SuiteCliTestCall) SetupSuite() {
 
 	s.testAddress, err = contracts.CalculateAddress(contracts.NameTest, 1, []byte{1})
 	s.Require().NoError(err)
-
-	zerostateCfg := fmt.Sprintf(`
-contracts:
-- name: Test
-  address: %s
-  value: 100000000000000
-  contract: tests/Test
-`, s.testAddress.Hex())
-
-	s.zeroState, err = execution.ParseZeroStateConfig(zerostateCfg)
-	s.Require().NoError(err)
-	s.zeroState.MainPublicKey = execution.MainPublicKey
 }
 
 func (s *SuiteCliTestCall) SetupTest() {
+	s.zeroState = &execution.ZeroStateConfig{
```

Comment:
You probably don't need a struct member anymore
```suggestion
	zeroState := &execution.ZeroStateConfig{
```